src/Marten.Testing/Linq/MatchesSql/MatchesSqlExtensionsTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Marten.Linq;
+using Marten.Linq.MatchesSql;
+using Marten.Testing.CoreFunctionality;
+using Marten.Util;
+using Shouldly;
+using Xunit;
+
+namespace Marten.Testing.Linq.MatchesSql
+{
+    public class MatchesSqlExtensionsTests
+    {
+        [Fact]
+        public void Throws_NotSupportedException_when_called_directly()
+        {
+            ShouldThrowExtensions.ShouldThrow<NotSupportedException>(
```

Comment:
Can you just use `Should.Throw<T>` everywhere here to be consistent w/ the rest of the codebase? It's minor, but does help readability. Otherwise this is all good.

Reply:
Fixed it.